Test/ManyManySelectors.m

```diff
@@ -49,5 +50,6 @@ int main(void)
 	objc_msgSend([Test class], nextSel);
 	assert(methodCalled == YES);
 	return 0;
+#endif // __GNUSTEP_MSGSEND__
```

Comment:
This test uses both lookup paths, it would be good to test the one that works everywhere unconditionally

Reply:
I see